llvm/SUBMODULES.json

```diff
@@ -2,9 +2,9 @@
   {
       "name": "llvm", 
       "url": "git://github.com/mono/llvm.git",
-      "rev": "9f79399f87282524fee099b328bd8cbf07929daf",
-      "remote-branch": "origin/master", 
-      "branch": "master", 
+      "rev": "release_39",
```

Comment:
`rev` is a commit number, not a branch to be able to precisely point to a commit. Please put the specific commit of the tip of `release_60`.